### PR TITLE
Upgrade Newtonsoft.Json and NQuads parsing fixes

### PR DESCRIPTION
**Pull request by tpluscode**
Changes in Newtonsoft.Json >=6 break conversion from JSON-LD to RDF.
I'm also re-submitting the culture-specific fixes

### CODE REVIEW

**Comment by asbjornu** [comment]
@tpluscode As this pull request is close to 2 months old with no reaction from a human being and the repository hasn't been touched by a human for 1.5 years, how do you feel about my idea in #13 to fork this baby, so we can get things moving?

**Comment by tpluscode** [comment]
Yes, this is very disappointing turn of events. I'm not sure how this is going to work out. Would you publish an alternative package? Or just ship binaries with Nancy.RDF, possibly IL merged? Or maybe publish the package with same id on MyGet?

**Comment by asbjornu** [comment]
Publishing with the same ID is impossible, I think. Only maintainers of the existing package (@sblom and @nuget) can upload packages with the same ID. But we could either set up a public [MyGet](http://myget.org/) feed, mint a new ID or just package it with Nancy.Rdf.

A new ID and name for the forked repository could be `LinkedDataSharp`, `LinkedData.NET` or something similar.

File: src/JsonLD/Core/JsonLdProcessor.cs [hunk 1]
```diff
@@ -66,8 +66,25 @@ public static JArray Expand(JToken input, JsonLdOptions opts)
...
+
+            // 2) verification of DOMString IRI
+            bool isIriString = input.Type == JTokenType.String;
+            if (isIriString)
+            {
+                bool hasColon = false;
+                foreach (var c in ((string) input))
+                {
+                    if (c == ':')
+                        hasColon = true;
```
Comment (emgarten) [inline src/JsonLD/Core/JsonLdProcessor.cs]: style: add { }'s around this

**Comment by emgarten** [comment]
@tpluscode thanks for the PR, it's good to see this library getting some help :smile: 

The changes and extra tests look good to me. :shipit:

**Comment by asbjornu** [comment]
@emgarten Yay, a human! 😄  What's the chance of getting this PR merged and released on NuGet? Can you help out with that?

**Comment by emgarten** [comment]
@asbjornu I'll take a look at building the nupkg for this with the changes next week. @johnataylor is going to review this also.

**Comment by asbjornu** [comment]
@emgarten @johnataylor Brilliant! 😄  👍

**Comment by emgarten** [comment]
I've merged the changes and created a 1.0.5 package here: https://www.myget.org/F/jsonld/api/v2/package/JsonLD/1.0.5

@asbjornu @tpluscode would you try out the new package and verify that it works for you? If it does I'll push it to nuget.org.

**Comment by tpluscode** [comment]
@emgarten I've updated and it seems to work fine. 

Did you change the package name on purpose though? On NuGet it's **json-ld.net** and on MyGet it's **JsonLd**. I'd rather your didn't change that name.

**Comment by emgarten** [comment]
@tpluscode thanks for verifying and all the work on the PR, this is a great improvement :smile: 

I'll change before pushing this to nuget.org. I'm going to add a build script to this repo also to help with package creation and building.

**Comment by asbjornu** [comment]
@emgarten Brilliant, thanks!

**Pull request merged**